ex04.ex

```diff
@@ -75,10 +85,24 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
-
+  def even_odd( list ) do
```

Comment:
# +1 for using a cool abstraction (your `map_on_condition` function is commonly called `filter`)

# -1 for processing the input list twice